Firmware/RTK_Surveyor/menuFirmware.ino

```diff
@@ -615,9 +615,9 @@ void otaPullCallback(int bytesWritten, int totalLength)
     if (apConfigFirmwareUpdateInProcess == true)
     {
 #ifdef COMPILE_AP
-      char myProgess[50];
-      sprintf(myProgess, "otaFirmwareStatus,%d,", percent);
-      websocket->textAll(myProgess);
```

Comment:
Wow. My brain couldn't see that typo. Nice catch.

Reply:
"Glad to be of service!"
(More Marvin - obvs...)